### ISSUE TITLE
Installer experience is awkward on Mac

### ISSUE THREAD

**Issue by colemickens** [Azure/azure-cli#2679]
```
-- Created tab completion file at '/Users/redacted/lib/azure-cli/az.completion'
 
===> Modify profile to update your $PATH and enable shell/tab completion now? (Y/n): 
 
===> Enter a path to an rc file to update: 
 
===> Enter a path to an rc file to update: $HOME/test.rc
 
===> Enter a path to an rc file to update: 
```

This was confusing, I just gave up and ctrl-c'd. No idea why it couldn't just find a default rc, or create the temp one... or at least just let me skip this step... I'm not sure if there were other steps that were missed because I ctrl-c'd it.

**Comment by derekbekoe** [comment]
No `~/.bash_profile` or `~/.bashrc` file was found and the file given didn't exist so it kept on prompting you.

Agreed this experience could be improved.
Current suggestion is for the script to prompt the user with a Y/N prompt and create one of the above files automatically. This would be a best effort suggestion. Assuming a standard Mac set up, this would work fine.

This will be addressed as part of https://github.com/Azure/azure-cli/issues/2377#issuecomment-289527594.

**Comment by derekbekoe** [comment]
Temp workaround is
```
$ touch ~/.bash_profile
```

**Comment by colemickens** [comment]
@derekbekoe Once again, thanks for the super fast response!

For my own info, did we miss any setup steps by killing the install process at this point? I ask because I did this on a customer's machine while we were troubleshooting a private cloud scenario and I want to make sure they aren't stuck in any sort of bad/unsupported state as a result of ctrl-c'ing.

Otherwise, this can just be closed out in interest of whatever comes in #2377. Just was surprised by this today. I'm not really sure how that particular OS X env managed to not have `~/.bash_profile` or `~/.bashrc` anyway...

**Comment by derekbekoe** [comment]
Not set up steps were missed. When it gets to that point, the CLI is already installed!

Since it couldn't find the `~/.bash_profile` file though:
- `az` is not in the path so user must run `/Users/<username>/bin/az`
- There's not tab completion as that involved modifying the bash_profile file.

If the above is required, the user can just run the script again after creating the bash_profile file.
